meta-genivi-dev/meta-genivi-dev/recipes-dev-platform/images/genivi-dev-platform.bb

```diff
@@ -55,3 +55,5 @@ IMAGE_INSTALL_append_rpi = " \
 IMAGE_INSTALL_append_qemuall = " \
     connman-conf \
 "
+
+IMAGE_FSTYPES = "${CUSTOM_IMAGE_FSTYPE}"
```

Comment:
How about a comment here that CUSTOM_IMAGE_FSTYPE shall be defined in each machine-specific local.conf template.  (Logic being that if it is not a Yocto standard variable, it deserves a comment.)

Reply:
Sound good.